src/libraries/System.Formats.Tar/tests/TarReader/TarReader.TarEntry.ExtractToFile.Tests.Unix.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Formats.Tar.Tests
+{
+    public partial class TarReader_TarEntry_ExtractToFile_Tests : TarTestsBase
+    {
+        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS)] // https://github.com/dotnet/runtime/issues/68360
+        [Fact]
+        public async Task SpecialFile_Unelevated_Throws_Async()
+        {
+            using TempDirectory root = new TempDirectory();
+            using MemoryStream ms = GetTarMemoryStream(CompressionMethod.Uncompressed, TestTarFormat.ustar, "specialfiles");
+
+            TarReader reader = new TarReader(ms);
+            await using (reader)
```

Comment:
the new C# syntax for using does not support async using? that's a pitty

Reply:
Yes, it does. I was initially working on VS Code and I was getting errors saying that syntax wasn't supported. So I submitted changes like this one.

But then I started using Visual Studio 2022 again, and tried using the regular using syntax, and it works. So I joined most of the await usings like this one in my last commit, but missed a few others. I'll push another commit merging the rest of the await usings with the declaration of the disposable objects.